<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detecting_Image_Object</class>
 <widget class="QWidget" name="Detecting_Image_Object">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 127);</string>
  </property>
  <widget class="QPushButton" name="Browse">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(93, 93, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(124, 142, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Nature Images Recognition&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VGG16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(121, 111, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>VGG16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Custom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(98, 106, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>720</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Test Result&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Image_Shape">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>197</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(102, 102, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Image_Shape</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Inception">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(103, 111, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>InceptionV3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Best_Model">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>600</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(102, 112, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Best_Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VGG19">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(105, 107, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>VGG19</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Camera">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(90, 90, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_vgg16">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>670</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>vgg16</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_vgg19">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>690</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>vgg19</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_custom">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>710</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>custom</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_inception">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>730</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>inception</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResNet">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;ResNet&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(94, 105, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ResNet</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ResNet">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>750</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ResNet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Edge_Detection">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(103, 123, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Canny</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
